crypto/asn1/asn_mime.c

```diff
@@ -425,6 +425,8 @@ ASN1_VALUE *SMIME_read_ASN1(BIO *bio, BIO **bcont, const ASN1_ITEM *it)
             || hdr->value == NULL) {
             sk_MIME_HEADER_pop_free(headers, mime_hdr_free);
             ASN1err(ASN1_F_SMIME_READ_ASN1, ASN1_R_NO_SIG_CONTENT_TYPE);
+            sk_MIME_HEADER_pop_free(headers, mime_hdr_free);
```

Comment:
This is already done above.

Reply:
Yup, copy/paste error on my part... will update.